### DISCUSSION TITLE
Storybook Setup in Turbo Repo

### DISCUSSION THREAD

**Discussion by LinnJS** [vercel/turborepo#3252] [Help] [answered]
I was reading this guide on setting up storybook with turborepo and I noticed it did not have `transpile-modules` package to trasnspile types across packages and apps. Is this no longer needed? I am using Tailwind and Storybook and it seems I still need it just not sure why I need it. I thought it was a TS thing.

Here is a [link](https://turbo.build/repo/docs/handbook/tools/storybook#deploy-on-vercel) to the guide I am referencing.

**Comment by anthonyshew** [comment, marked as answer]
`next-transpile-modules` (now built-into Next.js as `transpilePackages`) is Next.js specific!
